cms/static/js/views/modals/edit_xblock.js

```diff
@@ -38,6 +38,15 @@ define(["jquery", "underscore", "gettext", "js/views/modals/base_modal", "js/vie
                 this.editOptions = options;
                 this.render();
                 this.show();
+                if (this.options.modalType == "html") {
+                    this.$('.editor-modes a[rel="modal"]').attr('title', gettext('This link will open in a modal window')).leanModal({
+                        overlay: 0.50,
+                        closeButton: '.action-modal-close'
+                    });
+                    this.$('.action-modal-close').click(function(e) {
+                        (e).preventDefault();
+                    });
```

Comment:
It's not a replacement, it's separately needed for the dynamically loading edit HTML modal